### PR TITLE
Add intel QvE multi thread test

### PR DESCRIPTION
**Pull request by asvrada**

### CODE REVIEW

File: tests/intel_qve_thread_test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ${PROJ}_enclave_key_pair DEPENDS ${PROJ}_enc_private.pem
+                                   ${PROJ}_enc_public.pem ${PROJ}_enc_pubkey.h)
+
+add_subdirectory(host)
+add_subdirectory(enc)
+
+add_test(
+  NAME tests/intel_qve_thread_test
+  COMMAND $<TARGET_FILE:${PROJ}> ${PROJ}_enc.signed
+          ${CMAKE_CURRENT_SOURCE_DIR}/data/tdx_quote tdx 2 10
```
Comment (radhikaj) [inline tests/intel_qve_thread_test/CMakeLists.txt:38]: Can you change to 2 seconds?

**Comment by asvrada** [comment]
bors try

File: tests/intel_qve_thread_test/enc/enc.conf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+#Enclave settings:
+Debug=1
+NumHeapPages=1024
```
Comment (radhikaj) [inline tests/intel_qve_thread_test/enc/enc.conf:6]: make 10000

File: tests/intel_qve_thread_test/enc/enc.conf [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+NumStackPages=1024
```
Comment (radhikaj) [inline tests/intel_qve_thread_test/enc/enc.conf:7]: 256

File: tests/intel_qve_thread_test/enc/enc.conf [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+NumTCS=16
```
Comment (radhikaj) [inline tests/intel_qve_thread_test/enc/enc.conf:8]: 32

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Review by mingweishih — APPROVED** [review]
LGTM.
Question: Is here still a race condition if calling oe_tdx_verifier_initialize with multi-threading?

**Comment by asvrada** [comment]
> LGTM. Question: Is here still a race condition if calling oe_tdx_verifier_initialize with multi-threading?

There will be. In another PR we will add lock to TDX init/shutdown

**Review by radhikaj — APPROVED** [review]
Bors r+

**Pull request merged**